Unqualified Dublin Core RDF.js

```diff
@@ -11,25 +11,50 @@
 		"dataMode": "rdf/xml"
 	},
 	"inRepository": true,
-	"lastUpdated": "2018-12-17 03:16:21"
+	"lastUpdated": "2019-06-11 13:33:25"
 }
 
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2018-2019 Simon Kornblith
```

Comment:
Did you speak with him? Any reason for the time spans here where Simon was not actively working on this?

Reply:
I haven't, no, but I can.

Reply:
This might not be needed, cf. https://github.com/zotero/translators/issues/951#issuecomment-145275715

Reply:
Then I simply suggest to change the date to the original creation date to indicate this is a older translator. Maybe we also should put the relicensing commit into a separate commit.

Reply:
Which date? The date in the license block? So just have 2018 in the license block?

Reply:
2008 should be okay

Reply:
done